benches/benches.rs

```diff
@@ -16,7 +17,7 @@ impl Actor for ChainLink {
 
     fn handle(
         &mut self,
-        context: &mut Context<Self::Message>,
```

Comment:
I like what you did with the context simplification!

Reply:
Well OTOH I introduce the responsibility for each actor to define the `type Context = ...`. I wish [associated type defaults](https://github.com/rust-lang/rust/issues/29661) were stable, but it doesn't look even close.